pydra/engine/task.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         self,
         func: ty.Callable,
         output_spec: ty.Optional[BaseSpec] = None,
+        output_names=None,
```

Comment:
no `output_names` - if needed to be specified they should use output_spec

Reply:
see this example: https://github.com/nipype/pydra/blob/master/pydra/engine/tests/test_task.py#L27

Reply:
perhaps an example we should work towards is how to annotate an existing function via the `to_task` decorator

Reply:
I was thinking that `output_names` could be the easier option

Reply:
i agree it would be easier, but we want them to do it through proper annotation. since we will want people to provide type hints as well eventually.

so we should see if we can add `__annotations__` to the function using the decorator and adding arguments to the decorator.

Reply:
ok, will work on it